utils/installer/uninstall.sh

```diff
@@ -2,3 +2,4 @@
 
 rm -rf .local/share/lunarvim
 sudo rm /usr/local/bin/lvim
+rm -rf .local/share/applications/lvim.desktop
```

Comment:
while at it, can you make these `~/.local` instead of `.local`

Reply:
done!